app/javascript/src/scenes/SettingsPage/scenes/teams/components/TeamsDataTable.jsx

```diff
@@ -44,47 +44,61 @@ class TeamsDataTable extends Component {
     );
   }
 
+  renderShowsTotal(start, to, total) {
+    return (
+      <span>
+        Showing { start } to { to } of { total } entries.
```

Comment:
Please put this string in app/javascript/src/config/locales/messages.js and use FormatedMessage component